frontend/src/metabase/search/components/SearchResult.jsx

```diff
@@ -97,18 +101,33 @@ function ItemIcon({ item, type }) {
   );
 }
 
+const CollectionBadgeRoot = styled.div`
```

Comment:
Would definitely encourage us to create a dir just for this component and a styled file for these styled components.

Reply:
Would also suggest to make a followup PR to keep the diff minimal

Reply:
Sure.